test/unit/routes/test-overview.js

```diff
@@ -192,3 +195,26 @@ describe('Overview csv export route', function () {
       })
   })
 })
+describe(`GET ${INDEX_URI}`, function () {
+  it('should respond with a 200', function () {
+    return supertest(app)
+      .get(INDEX_URI)
+      .expect(200)
+  })
+})
+
+describe(`GET ${INDEX_URI + '?HPP&TRUE=1/0'}`, function () {
+  it('should respond with an error', function () {
+    return supertest(app)
+      .get(INDEX_URI + '?HPP=TRUE/0')
+      .expect(500)
+  })
+})
+
+describe('/overview can be omitted', function () {
+  it('should respond with 200 when overview is omitted from the URL', function () {
+    return supertest(app)
+      .get(VALID_URL_WITHOUT_OVERVIEW)
+      .expect(200)
+  })
+})
```

Comment:
Do you need to add any new tests to test-court-reports-overview?

Reply:
Yea, I'll push some now.